manifests/child/apache.pp

```diff
@@ -31,7 +39,7 @@
     ssl                    => true,
     ssl_cert               => $ssl_cert,
     ssl_key                => $ssl_key,
-    ssl_ca                 => $ssl_ca,
+    ssl_ca                 => $_ssl_ca,
```

Comment:
Doesn't this also work:
```suggestion
    ssl_ca                 => pick($ssl_ca, $::pulp::ca_cert, $pulp::child::server_ca_cert),
```

Reply:
If I remember correctly there was one scenario which broke the tests if I did it that way.